Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu/mysql:8.0-20.04_beta
+
+ENV CONFIG_VERSION="0.8.22-1"
+
+RUN set -eux && \
+	apt-get update && \
+	apt-get install -y wget lsb-release gnupg
+
+RUN set -eux && \
+	wget -q https://dev.mysql.com/get/mysql-apt-config_${CONFIG_VERSION}_all.deb \
+        -O /tmp/mysql-apt-config_${CONFIG_VERSION}_all.deb && \
+    DEBIAN_FRONTEND=noninteractive dpkg -i /tmp/mysql-apt-config_${CONFIG_VERSION}_all.deb && \
+    DEBIAN_FRONTEND=noninteractive apt-get update && \
+    DEBIAN_FRONTEND=noninteractive apt-get install -y mysql-shell && \
+    rm -rf /var/lib/apt/lists/*
+
+EXPOSE 3306 33060
+
+COPY docker-entrypoint.sh /
+
+ENTRYPOINT ["/bin/bash", "/docker-entrypoint.sh"]
```

Comment:
IIRC there is a fairly comprehensive entrypoint in the upstream image, is there a reason we're overriding that with a much more naive implementation?

Reply:
embarrassingly, this entrypoint is a remnant of testing since the container kept exiting (as the upstream entrypoint initializes mysql data dirs, changes root user password and then shuts down). deleted in [69ec1f1](https://github.com/canonical/mysql-container/pull/1/commits/69ec1f179f718412815e44d05e7e846f96774445)